app/connectors/SecureMessageConnector.scala

```diff
@@ -32,4 +34,19 @@ class SecureMessageConnector @Inject()(httpClient: HttpClient, servicesConfig: S
     httpClient.GET[List[ConversationHeader]](
       s"$secureMessageBaseUrl/secure-messaging/conversations/HMRC-CUS-ORG/EORINumber")
 
+  def getConversation(clientName: String, conversationId: String)(
+    implicit ec: ExecutionContext,
+    hc: HeaderCarrier): Future[Conversation] =
+    httpClient.GET[Conversation](
+      s"$secureMessageBaseUrl/secure-messaging/conversation/$clientName/$conversationId/HMRC-CUS-ORG/EORINumber")
+
+  def recordReadTime(client: String, conversationId: String)(
+    implicit ec: ExecutionContext,
+    hc: HeaderCarrier): Future[HttpResponse] = {
+    val dateTime = ReadTime(DateTime.now)
+    httpClient.POST[ReadTime, HttpResponse](
+      s"$secureMessageBaseUrl/secure-messaging/conversation/add-read-time/$client/$conversationId",
```

Comment:
The endpoint of this has changed to => ` /conversation/:client/:conversationId/read-times`